app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -1,26 +1,36 @@
 <%
   id_prefix ||= "radio-#{SecureRandom.hex(4)}"
   items ||= []
+  is_conditional = false
 %>
+<% if items.any? %>
+<% items.each do |item| %>
```

Comment:
I think the indentation of this file could be improved, and some lines spaces could help break it up a bit.

Reply:
Fixed indentation